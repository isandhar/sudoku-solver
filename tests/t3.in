*E0*3FB**4*86**7
****A*E2**C***F*
**62**95A*D****8
**D***6*F3*****C
**F*7*C*0*5**E4A
DAB7***E1C3*F**6
EC3*D*4********2
*8****A****D**5B
****15*49FB78**E
****E****D*2****
7*A***F*CE*****9
***6*C*98****FB*
F6*C5B***9****8*
B*4E**DF7*A*****
*12**8**B**0**9*
***9***A3**5*C7*
