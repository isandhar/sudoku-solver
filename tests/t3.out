CE0A3FB1249865D7
871BADE250C649F3
3F62C495A7D10BE8
59D48760F3EB12AC
62F173C80B59DE4A
DAB7925E1C34F806
EC35D04B6A8F9712
4890F1A6E27DC35B
23CD15049FB78A6E
9B5FEA874D6231C0
70A8B6FDCE135429
14E62C39850A7FBD
F67C5B13D92EA084
B54E09DF78AC2631
A123487CB6F0ED95
0D896E2A3145BC7F
